src/react-apps/applications/runtime/src/shared/resources/language/fetch/fetchLanguageSagas.ts

```diff
@@ -1,6 +1,7 @@
 import { SagaIterator } from 'redux-saga';
 import { call, takeLatest } from 'redux-saga/effects';
-import { get } from '../../../../utils/networking';
+// import { get } from '../../../../utils/networking';
```

Comment:
Leave commented out until language is implemented in the new app-template?

Reply:
Yes.